### ISSUE TITLE
[Screen Reader- Dual Write- Customize already created Entity] Narrator does not announce position of each menu items after selecting any Entity.

### ISSUE THREAD

**Issue by Saurabh19Aug** [OfficeDev/office-ui-fabric-react#11519]
Bug copied from Dual write : https://msazure.visualstudio.com/OneAgile/_workitems/edit/5741720

**#URL** :https://tip.dataintegrator.trafficmanager.net/dualWrite?axenv=focdstest1envce1329c483fdc3a3devaos.cloudax.dynamics.com

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. 
“HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.

Please reach out to C&AI Teams channel for any process related queries.

**User Experience:** 

Screen reader users get impacted if position information about menu items is not provided via screen reader and users find difficult to get the proper information of menu items with their respective position.
Note: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch the URL:https://tip.dataintegrator.trafficmanager.net/dualWrite?axenv=focdstest1envce1329c483fdc3a3devaos.cloudax.dynamics.com
2. Sign in with Valid credentials.

URL: https://tip.dataintegrator.trafficmanager.net/dualWrite?axenv=focdstest1envce1329c483fdc3a3devaos.cloudax.dynamics.com
2. Sign in with Valid credentials.
3. Dual Write entity Maps screen appearing.
4. Navigate to any entity and select context menu button.
5. Navigate to "Entity mapping" and select it .
6. Navigate to the menu items present on the top of the page.
7. Verify the issue whether screen reader pass the position information or not.
                                  
**Actual Result:**
Narrator does not announce position of each menu items after selecting any Entity. Narrator only read the name of the menu items without position.
 
**Expected Result:**

Narrator should pass the complete information the menu items with their respective position i.e. 1 out off 3 so on.

**Note:** 
1. Same issue does not exist with chrome.
2. Similar issue exist throughout the application.

**MAS Reference:** 
MAS 4.2.1 - Object Information (33A)
[MAS4.2.1-Customize already created Entity-Narrator does not announce position of each menu items.zip](https://github.com/OfficeDev/office-ui-fabric-react/files/3982597/MAS4.2.1-Customize.already.created.Entity-Narrator.does.not.announce.position.of.each.menu.items.zip)

**Comment by ecraig12345** [comment]
Can you please provide an isolated reproduction of this bug using codepen (http://aka.ms/fabricpen)? Without seeing how you're using the control, it's hard to say what's going wrong, and whether it's on the Fabric side or just an issue with how the control is being used. As it is, I can't even tell which Fabric control you might be talking about.

Also, **please** remember that this is a public forum and refrain from posting private issue links in the future! If you need to provide more context, you can file the issue and wait for it to be assigned, then contact the assignee privately on Teams.
